node/alephium-wormhole-9be054708cef-test-only.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
+  "project_id": "alephium-wormhole",
+  "private_key_id": "9be054708cefbaedb52b6e34251f417b84a2a8d4",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQCUJjOCSS4tlsao\nue67MZp/XMZcybgjzdEOMLsWglpOMGWVSq1tiUVOkyqYB3vwEPLP5EH9E4abGAoR\nwvf+LvWl5PfzAIyjAkS7kb1lY0+6mNzv8qnnVDtKcxlA2mAabGyiPzuKPNt1ZM8U\nm9qy/guSCtgnQ6cAI9b+GgK06+taLXUiJhIoFDlYRAYvEQUsNAgfzamH4FDIcmIG\nPsyvwVY5B+ncpOtZgWferiqVe91EyEVlehHJm2hWDVJWyh+c+B9lf1qixdn7y/g2\nwRBH7g89ijOGZYA3oREXJa/8WiMp2bqef0PK0XIUes6kP9Xkm+pWOkNo1KcXCo6C\nwwG5cZixAgMBAAECggEABMctTBtoxjGhsJlmjgUDMu5eu2lfa1AyWN8mJDYKDDww\nBTObtPeuE42thv+0IJb7WHcdJbi2flQKfG4pqg9WVDEL9NC6DcZ/SalhTrQi7dAL\nosrUbfs1JP6MpvR5ZatviYAOlIkdRC5Ue26stq8jgmSfz2G1hiH3HMo2oiJ7EbUZ\ns+wphShXs64VL4Z3OlOlfB+F/GOhdQODqsCqGqGrSHvTsjftXz8xYI78Js4KdtBX\n9BAYCXlH3XDsQB/7TbONsdxQMueyN00zHFBubbGYvJ4C3bIrFShw0O1Dx/KTy7uc\nay/qYyeGfWjaGR9JLGaP0zjqompZHDbfDQzXLyHjPQKBgQDMlN93EcQY+Jc0BU/F\n4rKYcvZ8Lx2wrBWbWOS1VDjlsQQt290S1RaIUgfW+xomswduqMJbUktyn2uDGLUe\nli2T4n8lc+IkrvV42V1+zGqmXPOJbwodCak89szmaYoFQ3RvJkyP4SBkn2R9ecdt\nZg6CatDQYJ8ejW9GIGbh46ALdwKBgQC5YmBj8L49CQWn8N9QY8NEBTbYk2MAml4Q\n+FLCyEhhFzONuyTVBi+crs0UH4F+beK9QOPy1Z+nYntU8wrb7Cmx0K0IqqSXgXkl\nwc9FpftX+c+xMRw7VUR/n4y9zKnLV3ghWGzPPIFqhevdYbF/vw8N1cNqeSwvIDJx\nxnSka7Y3FwKBgGWqNcK+GW0vXDSFzzbPiWEpAESGRYLegX5Jlp5l1ECQzzra3+nT\n6vajjTHQZxaEMVVUmAE5zDSR9KOZS6h/iPbMv9Kk2+TO7T25BzVT8EvsNTo+7A+y\nFWAuZXS7TdN/ygTSS4nUnnximVWhhvFenidBXbOv0GMmdf2rbHUk4QvRAoGACtEi\nrrItZ4JPQi1b2mT3z9A+EwUGkdJwCq1ypsro7kFcULjKvWNPA+VvMe1ksOvYfxJ6\nOoEcf7fPWnWT/KEbDbUPY6IRWIXFe96DKZ01vTxmUgB7lTq0dH3qyGhYv/Sqncd7\nEdyF8KE/0fsRUFLlnfkLHftUCKvoRZs+0xk12KECgYBDPJWmLdBuBHrK2PE7T8Y1\n6u5ld6ajYQGjMozsAJdpVTU6SFq3YC4miSbl7b2fW/xONMH+Vp6HsIheif7ek3L4\nALlBPmqnsHXFEmxvMyVx1kI17cwF10Wbhsn46xW/jqTPLIqUWdh6H6qYSj8hznNp\nkocMn4n8eL0MmA6sdlsLLg==\n-----END PRIVATE KEY-----\n",
+  "client_email": "guardian-testing@alephium-wormhole.iam.gserviceaccount.com",
+  "client_id": "117855719932263415086",
+  "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+  "token_uri": "https://oauth2.googleapis.com/token",
+  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+  "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/guardian-testing%40alephium-wormhole.iam.gserviceaccount.com"
```

Comment:
Just to confirm: is this a fake credential?

Reply:
Yes, this is a credential that only has the permission to work with a test KMS key.
The service account is generated [here](https://github.com/alephium/alephium-ops/blob/b9ffbb253d3c159c172dcce08dc0a0057a54ed18/bridge/terraform/kms-for-testing.tf#L2)

Reply:
This is dangerous in multiple ways:
1. this file plus the key name is leaking quite a lot information about the setup
2. it adds an dependency on the proper setting of infra and account setup, which might go wrong due to maintenance or upgrade.

I will think about how to mitigate this.